react-props-and-expressions/my-app/src/Btn-component.js

```diff
@@ -0,0 +1,7 @@
+export default function MyButton(text, style) {
```

Comment:
By convention we would name the file exactly the same as the component it exports.